code/modules/ruins/lavalandruin_code/techcult.dm

```diff
@@ -0,0 +1,592 @@
+	prereq_ids = list("adv_robotics", "emp_super", "mars_tech")
+	design_ids = list("borg_chest_empproof", "borg_head_empproof", "borg_l_arm_empproof", "borg_r_arm_empproof", "borg_l_leg_empproof", "borg_r_leg_empproof")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
+	export_price = 5000
+
+/datum/design/borg_chest/emp
+	name = "Cyborg Torso"
+	id = "borg_chest_empproof"
+	build_type = PROTOLATHE | MECHFAB
+	build_path = /obj/item/bodypart/chest/robot/empproof
+	materials = list(/datum/material/iron=40000, /datum/material/plasma=20000, /datum/material/uranium=10000)
+	construction_time = 150
+	category = list("Misc", "Medical Designs")
+
+/datum/design/borg_head/emp
+	name = "Cyborg Head"
+	id = "borg_head_empproof"
+	build_type = PROTOLATHE | MECHFAB
+	build_path = /obj/item/bodypart/head/robot/empproof
+	materials = list(/datum/material/iron=5000, /datum/material/plasma=2500, /datum/material/uranium=1250)
+	construction_time = 150
+	category = list("Misc", "Medical Designs")
+
+/datum/design/borg_l_arm/emp
+	name = "Cyborg Left Arm"
+	id = "borg_l_arm_empproof"
+	build_type = PROTOLATHE | MECHFAB
+	build_path = /obj/item/bodypart/l_arm/robot/empproof
+	materials = list(/datum/material/iron=10000, /datum/material/plasma=5000, /datum/material/uranium=2500)
+	construction_time = 100
+	category = list("Misc", "Medical Designs")
+
+/datum/design/borg_r_arm/emp
+	name = "Cyborg Right Arm"
+	id = "borg_r_arm_empproof"
+	build_type = PROTOLATHE | MECHFAB
+	build_path = /obj/item/bodypart/r_arm/robot/empproof
+	materials = list(/datum/material/iron=10000, /datum/material/plasma=5000, /datum/material/uranium=2500)
+	construction_time = 100
+	category = list("Misc", "Medical Designs")
+
+/datum/design/borg_l_leg/emp
+	name = "Cyborg Left Leg"
+	id = "borg_l_leg_empproof"
+	build_type = PROTOLATHE | MECHFAB
+	build_path = /obj/item/bodypart/l_leg/robot/empproof
+	materials = list(/datum/material/iron=10000, /datum/material/plasma=5000, /datum/material/uranium=2500)
+	construction_time = 100
+	category = list("Misc", "Medical Designs")
+
+/datum/design/borg_r_leg/emp
+	name = "Cyborg Right Leg"
+	id = "borg_r_leg_empproof"
+	build_type = PROTOLATHE | MECHFAB
+	build_path = /obj/item/bodypart/r_leg/robot/empproof
+	materials = list(/datum/material/iron=10000, /datum/material/plasma=5000, /datum/material/uranium=2500)
+	construction_time = 100
+	category = list("Misc", "Medical Designs")
+
+/***************** T4 cybernetics *****************/
+
+//Heart
+
+/obj/item/organ/heart/cybernetic/tier4
+	name = "quadro-cybernetic heart"
+	desc = "An electronic device designed to mimic the functions of an organic human heart. Can inject with set of chemicals when user enter critical condition and regenerate the dose later."
+	icon = 'icons/Fulpicons/tier4_organs.dmi'
+	icon_state = "heart-c-u3"
+	maxHealth = 3 * STANDARD_ORGAN_THRESHOLD
+	dose_available = TRUE
+	emp_vulnerability = 5
+
+/obj/item/organ/heart/cybernetic/tier4/on_life()
```

Comment:
shouldn't this already ben inherited from normal cybernetic hearts?

Reply:
This proc is on tier2 and tier 3 only, **as far as I know.**
Additionally, this one is a bit different.